src/test/java/PresenterTest.java

```diff
@@ -22,42 +28,45 @@ public String readLastSavedInput(){
 public class PresenterTest {
 
     @Test
-    public void whenTheServerStartsShowAwaitingInputMessage() throws IOException{
+    public void whenTheServerStarts_ShowAwaitingInputMessage() throws IOException {
```

Comment:
Additionally, with a generic logging interface, we won't need to add a new test every time we have a new message log.